torch/_inductor/pattern_matcher.py

```diff
@@ -50,8 +50,13 @@
 Constant = Any
 NodeOrConstant = Union[Constant, torch.fx.Node]
 
+
+class Multiple:
+    pass
+
+
 # Sentinel indicating multiple quantities can be matched
-MULTIPLE = object()
+MULTIPLE = Multiple()
```

Comment:
why was this needed, just curious ?

Reply:
Just for typechecking reasons. It woulda caught the prettyprinter error (although I guess I'm not actually using the prettyprinter here haha).